src/comodo/mujocoSimulator/mujocoSimulator.py

```diff
@@ -26,6 +32,16 @@ def load_model(self, robot_model, s, xyz_rpy, kv_motors=None, Im=None):
         self.kv_motors = (
             kv_motors if not (kv_motors is None) else np.zeros(self.robot_model.NDoF)
         )
+        self.H_left_foot = copy.deepcopy(self.robot_model.H_left_foot)
+        self.H_right_foot = copy.deepcopy(self.robot_model.H_right_foot)
+        self.H_left_foot_num = None 
+        self.H_right_foot_num = None 
+
+    def get_contact_status(self):
+        left_wrench, rigth_wrench = self.get_feet_wrench()
+        left_foot_contact = left_wrench[2] > 5
+        right_foot_contact = rigth_wrench[2] > 5
```

Comment:
5 is a magic number? 😊

Reply:
Improved in 6090ad00c61c867ae6d8e7e0954bdcb4aa298d6a (dependend on the robot mass )